css/global.css

```diff
@@ -0,0 +1,3 @@
+iframe {
+    opacity: 1 !important;
```

Comment:
This one line of code reveals the hidden iframes. 

It may be simple, but its simplicity in some ways makes it harder to bypass.